components/server/src/auth/authenticator.ts

```diff
@@ -110,6 +110,15 @@ export class Authenticator {
             res.redirect(this.getSorryUrl(`Bad request: missing parameters.`));
             return;
         }
+        // Logins with organizational Git Auth is not permitted
+        if (authProvider.info.organizationId) {
+            log.info({ sessionId: req.sessionID }, `Login with "${host}" is not permitted.`, {
+                "authorize-flow": true,
+                ap: authProvider.info,
+            });
+            res.redirect(this.getSorryUrl(`Login with "${host}" is not permitted.`));
```

Comment:
Is there a better call to action we can include? What should the user do when this happens?

Reply:
Great question. 
The response here is what's done on bad requests in that area. The legacy sorry page is used as a sink where we're lacking proper rending on the dashboard.
Why calling that a bad request? Those organizational Git Auth providers won't be available on the Login screen, they are not meant to be included there. Here we are guarding from forged requests to the `/api/login` endpoint parameterized with the `host` of an organizational Git Auth provider.
Given that one cannot reach this point by navigating any link, I don't think it make sense to provide any CTA. But that's definitely biased, so if find anything helpful, let's do that.

Reply:
I'm ok with this, given that it's a not a case a user can end up with in a regular case.

:cloud: To better highlight this, we could also just return a `400` (Bad Request). But I think it's more cosmetics.